.github/actions/nf-test-action/action.yml

```diff
@@ -56,8 +60,11 @@ runs:
       if: env.SENTIEON_ENCRYPTION_KEY != '' && env.SENTIEON_LICENSE_MESSAGE != ''
       shell: bash
       run: |
+        python -m pip install cryptography
         nextflow secrets set SENTIEON_AUTH_DATA $(python3 tests/modules/nf-core/sentieon/license_message.py encrypt --key "$SENTIEON_ENCRYPTION_KEY" --message "$SENTIEON_LICENSE_MESSAGE")
 
+      # TODO If conda-fail.yml exists and matrix = conda skip
+      # Really we need in-verse tag selection https://github.com/askimed/nf-test/issues/260
```

Comment:
while we wait for that, maybe we can add it here: https://github.com/adamrtalbot/detect-nf-test-changes/pull/12